ansible/roles/set-hostname/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+- name: Update /etc/hostname
+  become: yes
+  when: new_hostname is not undefined
+  copy:
+    content: "{{ new_hostname }}"
+    dest: /etc/hostname
+  register: hostname_changed
+
+- name: Update /etc/hosts
+  become: yes
+  when: new_hostname is not undefined
+  lineinfile:
+    path: /etc/hosts
+    regexp: '127.0.1.1	raspberrypi'
```

Comment:
Is this a regex string? If so, should it be something like `127\.0\.1\.1\s+raspberrypi`?
Also, is this supposed to be 127.0.0.1 or 127.0.1.1?

Reply:
nah it works the way it is, its copied directly from the pi itself.  if you leave this out, there isn't any practical effect except it throws a warning when you use sudo